app/views/answers/update.js.erb

```diff
@@ -0,0 +1,5 @@
+<% if @answer.errors.present? %>
+  $('.answer_errors').html('<%= j render 'common/errors', resource: @answer %>');
+<% else %>
+  $('.answers').html('<%= j render 'questions/answers', answer: @answer %>');
```

Comment:
This replaces the whole answers list with the only one answer. You should replace only the updated answer HTML.